### PR TITLE
Add SSI blog

### PR DESCRIPTION
**Pull request by r-j-arnold**
### Summary

I (@r-j-arnold) wrote a blog for the SSI about the Turing way. Archive a copy on the repo.

### List of changes proposed in this PR (pull-request)

Add the blog to the project management folder

### What should a reviewer concentrate their feedback on?

Is it in the right place? I've put it in the project management folder because that's where the newsletters are.

### Acknowledging contributors

By submitting this pull request I confirm that:

- [x] all contributors to this pull request who wish to be included are named in [humans.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/humans.md).

### CODE REVIEW

File: project_management/SSI_blog_06_03_19.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+If you have tips and tricks, or “gotchas” that caught you out as you tried to make your work reproducible, please share them through [this google form](https://docs.google.com/forms/d/e/1FAIpQLSfcyYH_E03Y5zuBdwrikQP4QldqzKmD-aPngCKthdV9e9alaA/viewform).
+
+## Workshops for Researchers and Research Software Engineers
+
+We’re running some workshops as part of the project. Two (one in Manchester on March 1st and one in [London on March 12th](https://www.eventbrite.co.uk/e/boost-your-research-reproducibility-with-binder-london-registration-55337162944)) was/will be on helping people to boost their research reproducibility with [Binder](https://mybinder.readthedocs.io/en/latest/). Binder is a tool for helping people share code without worrying about the computational environment they're running or installing a long list of requirements.
```
Comment (KirstieJane) [inline project_management/SSI_blog_06_03_19.md]: Has this blog done out yet? Does this section need to be re-written in the past tense and to include some feedback from the Manchester event?

Comment (r-j-arnold) [reply project_management/SSI_blog_06_03_19.md]: Yes it's out. I've included the "was/will"s for Manchester/London

**Comment by r-j-arnold** [comment]
Moved it to communications/blogs. Merging now.

**Comment by KirstieJane** [comment]
Fab! Thank you @r-j-arnold

File: project_management/SSI_blog_06_03_19.md (hunk 1 above)
Comment (KirstieJane) [reply project_management/SSI_blog_06_03_19.md]: Gotcha! Sorry - I missed the link on the top line.

Comment (r-j-arnold) [reply project_management/SSI_blog_06_03_19.md]: No problem!

**Pull request merged**